src/e2e/step-definitions/zaak.ts

```diff
@@ -20,12 +20,12 @@ const TEST_PERSON_HENDRIKA_JANSE_PHONE_NUMBER = "0612345678";
 
 async function checkZaakAssignment(
   this: CustomWorld,
-  zaakNumber: any,
+  caseNumber: any,
```

Comment:
Well.. we agreed that English is our preferred language for source code. However we have so far made exceptions for the really ZGW/ZAC specific vocabulary, most importantly: zaken. I would stick with `zaakNumber` etc here. This is also because there is a difference between a zaak and a case. A zaak is really a very specific type of case you could say but it is not just any case for sure.

Reply:
Case is a very generic term which stems from for example CMMN. Zaak on the other hand is defined by the 'zaakgerichtwerken' GEMMA architecture and specifically by the ZGW APIs.

Reply:
Alright; then I will revert this, and change all other occurrences in this file along as well.

Reply:
Reverted